src/models/__tests__/helpers/mock.ts

```diff
@@ -0,0 +1,52 @@
+import { container } from 'tsyringe'
+import { RAW_ENV_TOKEN } from '../../../env/common'
+
+const requiredEnvs = {
```

Comment:
potentially could use https://github.com/digicatapult/veritable-ui/blob/5f1a9de5495ee33f4de2b8b24d2c187605eddb1d/src/env/common.ts#L13 instead? If it works...